README.md

```diff
@@ -147,6 +170,22 @@ _Example_:
     }
 ```
 
+_Example in Hiera_:
+
+```yaml
+firewalld::custom_services:
+  puppet:
+    short: 'puppet'
+    description: 'Puppet Client access Puppet Server'
+    port:
+      - port: 8140
+      - protocol: 'tcp'
```

Comment:
Just a small issue but it'll mess with anyone who cut'n'pastes this...

This is an array of hashes... it needs to produce

`{"port"=>[{"port"=>8140, "protocol"=>"tcp"}]}`

Your YAML will split port and protocol out into separate hashes.

`{"port"=>[{"port"=>8140}, {"protocol"=>"tcp"}]}`

You need to drop the "-" on this line so it reads....

```yaml
port:
  - port: 8140
    protocol: 'tcp'
```